### ISSUE TITLE
A very small enhancement to allow people to just directly run from the git repository

### ISSUE THREAD

**Issue by johan-boule** [rg3/youtube-dl#16625]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2018.06.02*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [X] I've **verified** and **I assure** that I'm running youtube-dl **2018.06.02**

### Before submitting an *issue* make sure you have:
- [X] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [X] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [X] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [X] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:

Add the `-v` flag to **your command line** you run youtube-dl with (`youtube-dl -v <your command line>`), copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-v']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2018.06.02
[debug] Git HEAD: c3023e9
[debug] Python version 2.7.12 (CPython) - Linux-4.13.0-43-generic-x86_64-with-neon-16.04-xenial
[debug] exe versions: none
[debug] Proxy map: {}
...
<end of log>
```

---

### Description of your *issue*, suggested solution and other information

Since python doesn't require compilation before you can run it, it would make sense to make a very small adjustment in your git repository so that you can start the program directly. Currently, the startup script in the bin directory fails to find the module directory that's at the root of the repository. It's very easy to fix, but requires someone to know a bit about Python. Or you need to know that you can actually directly run youtube_dl/__main__.py, which again, is easy *if* you know a bit about Python, and actually reveals that it's only a matter of documentation (prolly put the the README, huh, there are two README files, btw). My point is many users will want to just clone the git repository and use it as a simple way to keep the software up-to-date. I see no reason of going through very Python-specific ways of compiling or installing something that doesn't require any of that by the virtue of Python being designed exactly for that :)

**Comment by dstftw** [comment]
Did you even [read FAQ](https://github.com/rg3/youtube-dl#developer-instructions)?

**Comment by johan-boule** [comment]
Ok I see what you mean on the FAQ. That's not the way I asked the same question, because the binary file in the tarballs is a big no to me. The answer is related since it somehow reveals that this process can be bypassed altogether with __main__.py May I propose a pull request with a clarification of that question ?On Jun 3, 2018 15:18, "Sergey M." <notifications@github.com> wrote:Did you even read FAQ?

**Comment by johan-boule** [comment]
I'm offering to improve the FAQ if you are kind enough to reopened this ticket, which as I explained earlier, it not a duplicate.On Jun 3, 2018 15:19, "Sergey M." <notifications@github.com> wrote:Closed #16625.

**Comment by johan-boule** [comment]
I give up.
